tests/Session/test_commands.py

```diff
@@ -100,6 +104,61 @@ def testHelp(self):
 
         # reset standout
         sys.stdout = sys.__stdout__
+    
+    @patch('src.Session.commands.input', create=True)
+    def testOpenFoodStorage(self, mocked_input):
+        # grab print output
```

Comment:
You should have a test for calling open if there are no foodStorages available and test that a different print output is given